languages/elixir/exercises/concept/multiple-clause-functions/.docs/hints.md

```diff
@@ -0,0 +1,29 @@
+## General
+
+In Elixir's ['Getting Started Guide'][guide] there is a nice refresher about named functions.
+
+## 1. Make the response when the guess matches the secret number
+
+You can use a [guard][guard] to check if the numbers match.
+
+## 2. Make the response when the guess is greater than the secret number
+
+You can make use of a combination [multiple function clauses][multiple-fn-clauses] and [guards][guard].
+
+## 3. Make the response when the guess is less than the secret number
+
+You can make use of a combination [multiple function clauses][multiple-fn-clauses] and [guards][guard].
+
+## 4. Make the responses when the guess is one more or one less than the secret number
+
+You can make use of a combination [multiple function clauses][multiple-fn-clauses] and [guards][guard].
```

Comment:
I can see that.  I struggled with how much to say in this one since this exercise could be very easy with more, but I try to add a little.  I looked at the hints for `fsharp/pattern-matching` for some guidance for this which has very similar hints to this iteration.

Reply:
Yeah, I sometimes also struggle with the same issue. I'm not sure I have the right solution, but the suggestion by @angelikatyborska sounds good.